tests/assets/move-projects/smove-clean-all.sh

```diff
@@ -10,7 +10,11 @@ build_dir=(
     "move-basics"
     "signer-scripts"
 )
-bundle_dir=("using_stdlib_natives")
+bundle_dir=(
+    "testing-move-stdlib"
+    "testing-substrate-stdlib"
+    "using_stdlib_natives"
```

Comment:
A bit unrelated to this PR:

I think we can combine both `build_dir` and `bundle_dir` in the same variable and delete both in the same `for` loop.

Since we use the same `rm -rf` command for deleting both project binaries, it doesn't matter whether we built it with `smove build` or `smove bundle`.